docs/TabBarExample.less

```diff
@@ -0,0 +1,46 @@
+@import (reference) "../src/less/index.less";
+
+.TabBarExample {
+  .margin--top--2xl;
+  font-size: 0.875rem;
```

Comment:
true, might be worth adding a size definition for it. we do use .875rem and 14px quite a bit, but not firmly on one side or the other.
was thinking about this recently; 16px sometimes feels a tad too large for web app UI text. feels less of an issue for documentation/preamble text or for copy-heavy pages, but slightly off inside UI elements/controls. 

WDYT? i'm aware of my bias toward what i've used on other projects, so i'd defer to you on this one :)
definitely seen a trend toward larger in the last few years, but again that seems to be more focused on content-centric sites vs web apps.